torch/nn/modules/module.py

```diff
@@ -540,7 +540,7 @@ def to(self, *args, **kwargs):
         .. function:: to(memory_format=torch.channels_last)
 
         Its signature is similar to :meth:`torch.Tensor.to`, but only accepts
-        floating point desired :attr:`dtype` s. In addition, this method will
+        floating point or complex :attr:`dtype`s. In addition, this method will
         only cast the floating point parameters and buffers to :attr:`dtype`
```

Comment:
I think this should be:
`In addition, this method will only cast the floating point and complex parameters and ... ` because the parameters can be of complex dtype as well

Reply:
Also, I think we should update the examples below to add an example demonstrating that Linear supports complex parameters and dtype.